### Q-2. Consider a dataset containing the heights (in centimeters) of 1000 individuals. The mean height is 170 cm with a standard deviation of 10 cm. The dataset is approximately normally distributed, and its skewness is approximately zero. Based on this information, answer the following questions:
a. What percentage of individuals in the dataset have heights between 160 cm
and 180 cm?

b. If we randomly select 100 individuals from the dataset, what is the probability
that their average height is greater than 175 cm?

c. Assuming the dataset follows a normal distribution, what is the z-score
corresponding to a height of 185 cm?

d. We know that 5% of the dataset has heights below a certain value. What is
the approximate height corresponding to this threshold?

e. Calculate the coefficient of variation (CV) for the dataset.

f. Calculate the skewness of the dataset and interpret the result

In [1]:
#From the given data above 

N = 1000
mean = 170
std = 10

Ans- a) It is telling % between 160 & 180 which is 1 std from mean 

<font color=green>
So accoring to empirical rule about 68% of data falls into that range of 160 & 180 cm.

Ans -b) We can use center limit theorm which says, if sample is selected, it is slo most likely to perform normal distribution. we can perform the steps to find it

In [2]:
import numpy as np

In [6]:
n = 100
p_mean = 175
#to find more then 175 cm

#To find the standard error for 170 cm

std_sample = std/np.sqrt(n)

In [7]:
std_sample

1.0

In [8]:
z_score = (p_mean - mean)/std_sample

In [9]:
z_score

5.0

<font color=green>
Probablility for z-score more of 5 is almost 100% (according to z-score table)

Ans: c) For z score of 185 cm, it can be calculated by:

In [10]:
value = 185

In [11]:
z_score = (value-mean)/std

In [12]:
print(z_score)

1.5


<font color=green>
The z_score for 185 cm will be equalls to 1.5 

Ans: d) To find the thershold for 5% of the dataset, need to find z-score with probablity of 0.05, which is corresponding to -1.64

In [13]:
z_score = -1.64

In [14]:
value = (z_score*std) + mean

In [15]:
value

153.6

<font color=green>
The value corresponding to thersold for 5% will be 153.6 cm

Ans: e) Covariance is given by below formula

In [18]:
CV = (std/mean)*100
CV

5.88235294117647

<font color=green>
The co variance of the dataset is about 5.88%

Ans: e) As it is given, dataset is normally distributed, so that mean ~ median

<font color=green>
As mean will be equal (or near to median), skewness of the data will be equals to 0.